pipenv/utils.py

```diff
@@ -873,6 +892,17 @@ def find_windows_executable(bin_path, exe_name):
     return exec_files[0]
 
 
+def convert_file_uri_to_path(uri):
```

Comment:
What are we envisioning this function being used for in the future? I'm inclined not to add it until it has a use.

Reply:
uhhhh this is an artifact that is no longer needed.  Will dig up the axe.